maricon.py

```diff
@@ -2,3 +2,4 @@
 
 #this is your discord bot's code
 key = 'insert here'
+key = 'MzE3NDg2MTM5NzM4OTQ3NTk1.DUQdTg.QHmUzVI1RXY2MAW5LmOfsjNHAow'
```

Comment:
are you fucking retarded? you commited your secret key! i will remove this but fucking change it immediately